design/TitanOperator Cards.md

```diff
@@ -0,0 +1,30 @@
+##Titan Operator Cards:
+
+Titan's Attack is affected by RPM(~ +.5% damage/RPM)
+Titan starts combat with 25 RPM. This is the baseline. Start of turn the RPM moves closer to the baseline by 1 + 1per10delta from baseline.
+
+Titan's Spell Power is affected by Heat(~+2%/heat)
+Titan starts combat with 2 Heat. He overheats when he has 10 heat.
```

Comment:
Are there any reasons to not overheat out of control? Overheating looks like an always good strategy...